ci-operator/config/openshift/jenkins/master.json

```diff
@@ -0,0 +1,172 @@
+                "src": {
+                    "paths": [
+                        {
+                            "source_path": "/go/src/github.com/openshift/jenkins/test-e2e/.",
+                            "destination_dir": "."
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "from": "base",
+            "to": "bin-2-centos",
+            "dockerfile_path": "Dockerfile",
+            "inputs": {
+                "src": {
+                    "paths": [
+                        {
+                            "source_path": "/go/src/github.com/openshift/jenkins/2/.",
+                            "destination_dir": "."
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "from": "base",
+            "to": "bin-slave-base-centos",
+            "dockerfile_path": "Dockerfile",
+            "inputs": {
+                "src": {
+                    "paths": [
+                        {
+                            "source_path": "/go/src/github.com/openshift/jenkins/slave-base/.",
+                            "destination_dir": "."
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "from": "bin-slave-base-centos",
+            "to": "bin-slave-maven-centos",
+            "dockerfile_path": "Dockerfile",
+            "inputs": {
+                "src": {
+                    "paths": [
+                        {
+                            "source_path": "/go/src/github.com/openshift/jenkins/slave-maven/.",
+                            "destination_dir": "."
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "from": "bin-slave-base-centos",
+            "to": "bin-slave-nodejs-centos",
+            "dockerfile_path": "Dockerfile",
+            "inputs": {
+                "src": {
+                    "paths": [
+                        {
+                            "source_path": "/go/src/github.com/openshift/jenkins/slave-nodejs/.",
+                            "destination_dir": "."
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "from": "bin-slave-base-centos",
+            "to": "bin-agent-maven-35-centos",
+            "dockerfile_path": "Dockerfile",
+            "inputs": {
+                "src": {
+                    "paths": [
+                        {
+                            "source_path": "/go/src/github.com/openshift/jenkins/agent-maven-3.5/.",
+                            "destination_dir": "."
+                        }
+                    ]
+                }
+            }
+        },
+        {
+            "from": "bin-slave-base-centos",
+            "to": "bin-agent-nodejs-8-centos",
+            "dockerfile_path": "Dockerfile",
+            "inputs": {
```

Comment:
I prefer the single line patterns for these, just because this is a ton of whitespace and harder to read.

Reply:
10-4 will update